plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -772,9 +772,9 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    }
 
    void restart_speculative_block() {
+      fc_dlog(_log, "Restarting exhausted speculative block #${n}", ("n", chain_plug->chain().head_block_num() + 1));
```

Comment:
A comment can be added here to indicate this dlog is for integration test, so it won't be removed accidentally in the furture.

Reply:
Adde comment